src/index.html

```diff
@@ -1,12 +1,45 @@
 <!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+
+  <body class="calendar calendar---start-day-mon calendar---length-31">
```

Comment:
either use as mentor showed or do like this (to follow BEM recommendations)
```suggestion
  <body class="calendar calendar--start-day--mon calendar--month-length--31">
```